internal/services/azurestackhci/stack_hci_cluster_resource_test.go

```diff
@@ -86,6 +86,21 @@ func TestAccStackHCICluster_update(t *testing.T) {
 	})
 }
 
+func TestAccStackHCICluster_automanageConfigurationAssignment(t *testing.T) {
+	data := acceptance.BuildTestData(t, "azurerm_stack_hci_cluster", "test")
+	r := StackHCIClusterResource{}
+
+	data.ResourceTest(t, r, []acceptance.TestStep{
+		{
+			Config: r.automanageConfigurationAssignment(data),
+			Check: acceptance.ComposeTestCheckFunc(
+				check.That(data.ResourceName).ExistsInAzure(r),
+			),
+		},
+		data.ImportStep("automanage_configuration_id"),
```

Comment:
Can you add another step to this test that uses the `r.complete` config. I think we should test the scenario where the automanage configuration is removed from the cluster.

Reply:
Ok, code has been update with complete config (only diff is adding a tag). I add a removeAutomanageConfigurationAssignment template to test the update and read function.